website/templates/base.html

```diff
@@ -11,7 +11,7 @@
     <link rel="shortcut icon" href="{% static 'img/favicon.ico' %}" />
     <link rel="stylesheet" href="{% static 'css/site.min.css' %}" media="screen">
     <script src="{% static 'js/lib/zepto.min.js' %}"></script>
-
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
```

Comment:
Good point. I added it because Zepto didn't have the `animate()` method... but I found a way to make the smooth scroll work with what we had, so i removed `jQuery`. Thanks!